lib/activerecord_sqlserver_adapter/jdbc/type_converter.rb

```diff
@@ -0,0 +1,96 @@
+module ActiveRecordSqlServerAdapter
```

Comment:
Reminder to make a jdbc folder in `lib/active_record/connection_adapters/sqlserver/jdbc` and put all these files in there with the other namespace.